docs/developing.rst

```diff
@@ -54,6 +54,42 @@ In edx-platform/lms/envs/private.py and edx-platform/cms/envs/private.py:
         'MUST_BE_VERIFIED_TRACK': False
     }
 
+
+How do I setup `mfe-special-exam-lib` for local development?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`MFE special exam lib <https://github.com/edx/frontend-lib-special-exams/>`_ is a react library to support
+special exams in the MFE application learning.
+
+Make sure that `frontend-app-learning` is setup and running on your devstack.
+
+The special exam lib is installed as a dependency of MFE application learning.
```

Comment:
#NIT `The special exam lib is installed as a dependency of MFE application learning.` should be `The special exam lib is installed as a dependency of Learning MFE.`

Reply:
fixed